### DISCUSSION TITLE
Use Translations in tooltips

### DISCUSSION THREAD

**Discussion by PabloPomodoro** [saadeghi/daisyui#2907] [Ideas / feature requests]
Hi,

first of all thank you for the awesome UI library!

My issue is that I cannot put a translation (I'm using [transloco](https://github.com/jsverse/transloco)) in the data-tip attribute from the tooltip component.
As I'm also using Angular the compiler fails with this message:

```
Application bundle generation failed. [8.624 seconds]
X [ERROR] NG8002: Can't bind to 'tip' since it isn't a known property of 'div'. [plugin angular-compiler]

    src/app/components/welcome/welcome.component.html:25:46:
      25 │ ...oltip-bottom" data-tip="{{ 'welcome.card.tooltip' | transloco }}">
         ╵                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Not sure if you would consider this a bug; propably more of a feature request.

I'm looking forward to your response :)

In the meanwhile all the best to you!

**Comment by saadeghi** [comment]
I think you should ask about this in the Transloco repo.

**Reply by PabloPomodoro** [reply]
Thank you for your reply!

What I tried as a workaround is to have a variable set to the tooltip, e. g. `data-tip={{ stringVariable }}` which produces the same error, which mean something on daisyui's side has to be done as well I suppose.

But I will also reach out to the transloco team :)
